src/app/(private)/(dashboard)/community/create/CommunityCreateMemberPage.tsx

```diff
@@ -0,0 +1,326 @@
+        if (isSaving) {
+            return;
+        }
+        if (!isValid) {
+            console.log("invalid");
+            return;
+        }
+        setIsSaving(true);
+        setAlertMessage(null);
+        try {
+            const { message, pr_url } = await createMemberAction(input);
+            setAlertMessage({
+                title: `Modifications enregistrées`,
+                message,
+                type: "success",
+            });
+            setPRURL(pr_url);
+        } catch (e: any) {
+            // todo: sentry
+            console.log(e);
+            setAlertMessage({
+                title: "Erreur",
+                //@ts-ignore
+                message: e.response?.data?.message || e.message,
+                type: "warning",
+            });
+            setIsSaving(false);
+            if (e.errors) {
+                control.setError("root", {
+                    //@ts-ignore
+                    message: Object.values(e.errors).join("\n"),
+                });
+            }
+        }
+        document.body.scrollIntoView();
+        setIsSaving(false);
+    };
+
+    return (
+        <>
+            {!prUrl && (
+                <>
+                    <h1>Créer une fiche membre</h1>
+
+                    <div className="fr-grid-row fr-grid-row-gutters">
+                        <div className="fr-col-12 fr-col-md-12 fr-col-lg-12">
+                            <form
+                                onSubmit={handleSubmit(onSubmit)}
+                                aria-label="Modifier mes informations"
+                            >
+                                <fieldset
+                                    className="fr-mt-5v fr-mb-0v fr-fieldset"
+                                    id="identity-fieldset"
+                                    aria-labelledby="identity-fieldset-legend identity-fieldset-messages"
+                                >
+                                    <legend
+                                        className="fr-fieldset__legend"
+                                        id="identity-fieldset-legend"
+                                    >
+                                        <h2 className="fr-h3">Informations</h2>
+                                    </legend>
+                                    <div
+                                        className={fr.cx(
+                                            "fr-fieldset__element",
+                                            "fr-col-12",
+                                            "fr-col-lg-4",
+                                            "fr-col-md-4",
+                                            "fr-col-offset-lg-8--right",
+                                            "fr-col-offset-md-8--right"
+                                        )}
+                                    >
+                                        <Input
+                                            label="Prénom"
```

Comment:
Pas très important mais pour info tu peux récupérer la description zod avec un truc comme ça:
```suggestion
                                            label={createMemberSchema.shape.firstname.description}
```